File: app/src/main/java/ru/vitaliy/belyaev/wishapp/entity/analytics/action_events/appActionsEvents.kt
package ru.vitaliy.belyaev.wishapp.entity.analytics.action_events

import ru.vitaliy.belyaev.wishapp.entity.analytics.AnalyticsEvent

object InAppReviewRequestedEvent : AnalyticsEvent {

    override val name: String = "InApp Review Requested"

    override val params: Map<String, Any?> = emptyMap()
}

object InAppReviewShowEvent : AnalyticsEvent {

    override val name: String = "InApp Review Show"

    override val params: Map<String, Any?> = emptyMap()
}

File: iosApp/WishApp/Domain/Analytics/AppActionsEvents.swift
//
//  AppActionsEvents.swift
//  WishApp
//
//  Created by Vitaliy on 20.07.2023.
//

import Foundation

class InAppReviewRequestedEvent: AnalyticsEvent {
    
    var name: String = "InApp Review Requested"
    
    var params: [String : Any]?
}
